### PR TITLE
Convert templates from Slim To ERb

### PR DESCRIPTION
**Pull request by frankieroberto**
Converts a bunch of attachment-related templates from Slim format to ERb, ahead of some changes to these pages.

See [rationale for adopting ERb](https://github.com/UKGovernmentBEIS/beis-opss-psd/blob/master/doc/decisions/0001-use-erb-for-view-templates.md).

### CODE REVIEW

File: psd-web/app/views/documents/_document_card.html.erb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+
+<div class="govuk-grid-row govuk-!-padding-bottom-6">
+  <div class="govuk-grid-column-one-quarter">
+    <%= render "documents/document_preview", document: document, dimensions: "300x200", restricted: true %>
+  </div>
+
+  <div class="govuk-grid-column-three-quarters">
+    <h2 class="govuk-heading-m govuk-!-margin-bottom-1"><%= title %></h2>
+    <span class="govuk-hint  govuk-!-font-size-16">
```
Comment (edwardhorsford) [inline psd-web/app/views/documents/_document_card.html.erb:14]:
```suggestion
    <span class="govuk-hint govuk-!-font-size-16">
```

File: psd-web/app/views/documents/_metadata_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                 key: :title,
+                 form: subform,
+                 classes: "govuk-!-width-two-thirds",
+                 value: document.metadata["title"],
+                 label: { text: "Document title" } %>
+        <%= render "form_components/govuk_textarea",
+                 key: :description,
+                 form: subform,
+                 value: document.metadata["description"],
+                 attributes: { maxlength: 10_000 },
```
Comment (edwardhorsford) [inline psd-web/app/views/documents/_metadata_form.html.erb:27]: what's up with this maxlength?

File: psd-web/app/views/documents/_document_card.html.erb [hunk 3]
```diff
@@ -0,0 +1,45 @@
+<% restricted = !can_be_displayed?(document, parent, current_user) %>
+<% title = restricted ? document_restricted_title : document.title %>
+<% safe = document.metadata["safe"] %>
+<% document = document.decorate %>
```
Comment (slorek) [inline psd-web/app/views/documents/_document_card.html.erb:4]: Not a blocker, but we don't need to split Ruby code across multiple tags, it makes it very verbose. There's a few examples of this, this is just one:

```suggestion
<%
  restricted = !can_be_displayed?(document, parent, current_user)
  title = restricted ? document_restricted_title : document.title
  safe = document.metadata["safe"]
  document = document.decorate
%>
```

File: psd-web/app/views/documents/remove.html.erb [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+          <th class="govuk-table__header" scope="row">
+            URL
+          </th>
+          <td class="govuk-table__cell">
+            <%= @file.filename %>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+
```
Comment (slorek) [inline psd-web/app/views/documents/remove.html.erb:42]:
```suggestion
```

File: psd-web/app/views/documents/_metadata_form.html.erb (hunk 2 above)
Comment (frankieroberto) [reply psd-web/app/views/documents/_metadata_form.html.erb:27]: No idea!

File: psd-web/app/views/documents/_document_card.html.erb (hunk 3 above)
Comment (frankieroberto) [reply psd-web/app/views/documents/_document_card.html.erb]: Thanks, fixed.

**Pull request merged**